plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -213,6 +213,11 @@ static int cb_cloudwatch_init(struct flb_output_instance *ins,
         ctx->sts_endpoint = (char *) tmp;
     }
 
+    tmp = flb_output_get_property("profile", ins);
+    if (tmp) {
+        ctx->profile = (char *) tmp;
+    }
+
```

Comment:
since you used offsetof in the config map, you don;t need this. It will be set automatically. Test it.